.travis/install_ubuntu_dependencies.sh

```diff
@@ -18,7 +18,7 @@ set -ex
 sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
 sudo apt-get update -o Acquire::CompressionTypes::Order::=gz
 
-DEPENDENCIES="unzip make indent kwstyle libssl-dev tcpdump valgrind"
+DEPENDENCIES="unzip make indent kwstyle libssl-dev tcpdump valgrind lcov"
```

Comment:
While you're here do you want to add `m4 nettle-dev nettle-bin pkg-config` from https://github.com/awslabs/s2n/issues/751